recipes-kernel/linux/linux-imx_%.bbappend

```diff
@@ -2,11 +2,23 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 # override kernel source
 SRCBRANCH = "${@bb.utils.contains('SUBMACHINE', 'oppcharge', \
-             'v4.11.12-evacharge-se-dfs', 'v4.9.11', d)}"
+                    'v4.11.12-evacharge-se-dfs', \
+                    bb.utils.contains('MACHINE', 'tarragon', \
```

Comment:
Is there a reason for the extra 'MACHINE' check?

Reply:
Yes, since for EVAcharge platform, we want to keep v4.9.